webservices/args.py

```diff
@@ -753,3 +753,10 @@ def _deserialize(self, value, attr, data):
         description=docs.COMMITTEE_TYPE_STATE_AGGREGATE_TOTALS
     ),
 }
+
+auditfinding = {
```

Comment:
PEP8-nit:  In Python, if you're dealing with something that is represented by two separate words, as in "Audit Finding," we would typically name this with an underscore, e.g., `audit_finding`.